app/src/main/res/layout/activity_photo_editor.xml

```diff
@@ -31,7 +31,7 @@
             android:background="@drawable/cornered_edges"
             android:paddingStart="5dp"
             android:paddingEnd="5dp"
-            android:text="@string/save_current"
+            android:text="@string/save"
```

Comment:
There is not string named "save" in resources. That's why the build is failing

Reply:
Oh i didn't commit the strings.xml file